.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        emacs_version:
+          - 24.3
+          - 26.3
+          - 27.2
+          - 28.2
+          - snapshot
+    steps:
+    - uses: purcell/setup-emacs@master
+      with:
+        version: ${{ matrix.emacs_version }}
+
+    - uses: actions/checkout@v3
+    - name: Install dependencies
+      run: sudo apt install pandoc aspell
```

Comment:
Why pandoc and aspell?

Reply:
It is wrong copy and paste, I'll fix later